packages/adapters/src/adapters/Cornerstone3D/Length.ts

```diff
@@ -84,30 +89,11 @@ class Length {
             point1,
             point2,
             distance,
-            trackingIdentifierTextValue,
+            trackingIdentifierTextValue: this.trackingIdentifierTextValue,
             finding,
             findingSites: findingSites || []
         };
     }
 }
```

Comment:
why are we removing `isValidCornerstoneTrackingIdentifier`?

Reply:
it is used in OHIF extensively

Reply:
It is handled by the parent class in a more generic fashion.